src/TimingData.h

```diff
@@ -460,6 +460,9 @@ class TimingData
 	/** @brief The initial offset of a song. */
 	float	m_fBeat0OffsetInSeconds;
 
+	/** @brief The offset applied to the song's entire group */
```

Comment:
As above, we must be very explicit what the offset and where it applied to. This value which is already applied to the song and not one "to be applied by engine" as they have opposite sense